webapp/src/components/icon/icon.jsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+import {makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
+
+import {Svgs} from '../../constants';
+
+export default class Icon extends React.PureComponent {
+    static propTypes = {
+        useSVG: PropTypes.bool.isRequired,
+    }
+    render() {
+        const style = getStyle();
+
+        let icon = (ariaLabel) => (
+            <span aria-label={ariaLabel}>
+                <i className='icon icon-brand-zoom'/>
+            </span>
+        );
+        if (this.props.useSVG) {
+            icon = (ariaLabel) => (
+                <span
+                    aria-label={ariaLabel}
+                    style={style.iconStyle}
+                    dangerouslySetInnerHTML={{__html: Svgs.VIDEO_CAMERA}}
+                />
+            );
+        }
+        return (
+            <FormattedMessage
+                id='zoom.camera.ariaLabel'
+                defaultMessage='zoom camera icon'
+            >
+                {(ariaLabel) => icon(ariaLabel)}
+            </FormattedMessage>
+        );
+    }
+}
+
+const getStyle = makeStyleFromTheme(() => {
+    return {
+        iconStyle: {
+            position: 'relative',
+            top: '-1px',
+        },
+    };
+});
```

Comment:
Could you add a newline and the end of this file and `index.js` to avoid possible superfluous diffs in the future?